ldirectord/ldirectord.in

```diff
@@ -2740,7 +2767,7 @@ sub run_child
 			_check_real($v, $r);
 		}
 		$0 = "ldirectord $virtual_id";
-		sleep $checkinterval;
+		sleep $checkinterval + (random($$) * $checkintervalspread);
```

Comment:
As noted, should be rand(), and as $$ is the current PID you're multiplying checkintervalspread by what is often likely to be a very large integer, so this becomes something like sleep(a few hours). Which doesn't seem right?

Surely should just be
```
sleep $checkinterval + rand($checkintervalspread);
```
?